src/Estranged.Lfs.Adapter.Azure.Blob/AzureBlobAdapterConfig.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Estranged.Lfs.Adapter.Azure.Blob
+{
+    public class AzureBlobAdapterConfig : IAzureBlobAdapterConfig
+    {
+        public string ConnectionString { get; set; }
+
+        public string ContainerName { get; set; } = "gitlfs";
```

Comment:
I think this should be null - I don't think this is a good default

Reply:
Yeah you are right, I think this was a left over from some early manual testing, I'll remove it.